docs/scos/dev/feature-integration-guides/202307.0/glue-api/dynamic-data-api/install-the-data-exchange-api-inventory-management-feature.md

```diff
@@ -0,0 +1,106 @@
+
+**src/Pyz/Zed/DynamicEntity/DynamicEntityDependencyProvider.php**
+
+```php
+<?php
+
+namespace Pyz\Zed\DynamicEntity;
+
+use Spryker\Zed\Availability\Communication\Plugin\DynamicEntity\AvailabilityDynamicEntityPostCreatePlugin;
+use Spryker\Zed\Availability\Communication\Plugin\DynamicEntity\AvailabilityDynamicEntityPostUpdatePlugin;
+use Spryker\Zed\DynamicEntity\DynamicEntityDependencyProvider as SprykerDynamicEntityDependencyProvider;
+
+class DynamicEntityDependencyProvider extends SprykerDynamicEntityDependencyProvider
+{
+    /**
+     * @return array<\Spryker\Zed\DynamicEntityExtension\Dependency\Plugin\DynamicEntityPostUpdatePluginInterface>
+     */
+    protected function getDynamicEntityPostUpdatePlugins(): array
+    {
+        return [
+            new AvailabilityDynamicEntityPostUpdatePlugin(),
+        ];
+    }
+
+    /**
+     * @return array<\Spryker\Zed\DynamicEntityExtension\Dependency\Plugin\DynamicEntityPostCreatePluginInterface>
+     */
+    protected function getDynamicEntityPostCreatePlugins(): array
+    {
+        return [
+            new AvailabilityDynamicEntityPostCreatePlugin(),
+        ];
+    }
+}
+```
+
+{% info_block warningBox "Verification" %}
+
+Let's say you want to have a new endpoint `/dynamic-data/stock-products` to operate with data in `spy_stock_product` table in database.
+
+Based on the provided information, the SQL transaction for interacting with the `spy_stock_product` table through the API request via `dynamic-entity/stock-products` would be as follows:
+
+```sql
+BEGIN;
+INSERT INTO `spy_dynamic_entity_configuration` VALUES (1,'stock-products','spy_stock_product',1,'{"identifier":"id_stock_product","fields":[{"fieldName":"id_stock_product","fieldVisibleName":"id_stock_product","isCreatable":false,"isEditable":false,"validation":{"isRequired": false},"type":"integer"},{"fieldName":"fk_product","fieldVisibleName":"fk_product","isCreatable":true,"isEditable":true,"type":"integer","validation":{"isRequired": true}},{"fieldName":"fk_stock","fieldVisibleName":"fk_stock","isCreatable":true,"isEditable":true,"type":"integer","validation":{"isRequired": true}},{"fieldName":"is_never_out_of_stock","fieldVisibleName":"is_never_out_of_stock","isCreatable":true,"isEditable":true,"type":"boolean","validation":{"isRequired": false}},{"fieldName":"quantity","fieldVisibleName":"quantity","isCreatable":true,"isEditable":true,"type":"integer","validation":{"isRequired": true}}]}', '2023-07-29 12:15:13.0', '2023-07-29 12:15:15.0');
+COMMIT;
+```
+
+To obtain an access token follow [How to send a request in Data Exchange API](/docs/scos/dev/glue-api-guides/{{page.version}}/dynamic-data-api/how-to-guides/how-to-send-request-in-data-exchange-api.html)
+
+### Sending a `PATCH` request
+This request needs to include the necessary headers, such as Content-Type, Accept, and Authorization, with the access token provided.
+
```

Comment:
Can we remove some of the information and instead give reference to the howto guide? That is, I would suggest replacing lines 69-82 with the following text:
 "Let's say you want to have a new endpoint `/dynamic-data/stock-products` to operate with data in `spy_stock_product` table in database. Send the PATCH request to update the product stock data according to the guidelines in [How to send a request in Data Exchange] API (/docs/scos/dev/glue-api-guides/202307.0/data-exchange-api/how-to-guides/how-to-send-request-in-data-exchange-api.html):"
 
 What do you think?

Reply:
we don`t have information about the `stock-products` endpoint in the How To guide, so it looks like we can`t remove anything here (